packages/shared/src/hooks/feed/useTrackImpression.ts

```diff
@@ -46,6 +46,7 @@ export default function useTrackImpression(
                 isFeedbackEnabled && item.post.type === PostType.Article
```

Comment:
isFeedbackEnabled should always return true right?

Reply:
Modified, let me know what you think